src/Jira.js

```diff
@@ -86,12 +74,28 @@ export default class Jira {
       promises.push(
         this.getJiraIssue(id)
         .then((ticket) => {
-          if (includeTicket(ticket)) {
+          if (this.includeTicket(ticket)) {
             log.tickets.push(ticket);
+            return ticket;
           }
         })
         .catch((err) => {
-          console.log(`Ticket ${id} not found: ${err}`);
+          console.log('Ticket not found', id);
+        })
+        .then((ticket) => {
+
+          // Add release version
+          if (ticket && releaseVersion) {
+            ticket.fields.fixVersions.push({name: releaseVersion});
```

Comment:
Will Jira automatically prevent the same release version being listed on a ticket more than once? (just in case this ticket is already attached to this release)